src/legacy/server/index_patterns/mixin.ts

```diff
@@ -55,9 +72,10 @@ export function indexPatternsMixin(kbnServer, server) {
    *  @method request.getIndexPatternsService
    *  @type {IndexPatternsService}
    */
-  server.addMemoizedFactoryToRequest('getIndexPatternsService', request => {
+  server.addMemoizedFactoryToRequest('getIndexPatternsService', (request: Legacy.Request) => {
```

Comment:
The `addMemoizedFactoryToRequest` code doesn't seem to be present in any existing type definitions, what is it supposed to do?